python/amici/sbml_import.py

```diff
@@ -1107,3 +1109,34 @@ def checkLibSBMLErrors(sbml_doc, show_warnings=False):
         raise SBMLException(
             'SBML Document failed to load (see error messages above)'
         )
+
+
+def _check_unsupported_functions(sym):
+    """Recursively checks the symbolic expression for unsupported symbolic
+    functions
+
+        Arguments:
+            sym: symbolic expressions @type sympy.Basic
+
+        Returns:
+
+        Raises:
+            raises SBMLException if an unsupported function is encountered
+    """
+
+    unsupported_functions = [sp.factorial, sp.ceiling]
```

Comment:
Can we add `piecewise` here? Occurred in some models of the benchmark collection.